gulpfile.js

```diff
@@ -55,32 +48,8 @@ gulp.task(
   })
 )
 
-gulp.task(
-  'coveralls',
-  gulp.series('test', function coverallsTask() {
-    if (!process.env.CI) {
-      return Promise.resolve()
-    }
-
-    return gulp
-      .src(path.join(__dirname, 'coverage/lcov.info'))
-      .pipe(coveralls())
-  })
-)
-
 gulp.task('clean', function() {
   return del('dist')
 })
 
-gulp.task(
-  'babel',
-  gulp.series('clean', function babelTask() {
-    return gulp
-      .src('lib/**/*.js')
-      .pipe(babel())
-      .pipe(gulp.dest('dist'))
-  })
-)
-
-gulp.task('prepublishOnly', gulp.series('babel'))
```

Comment:
Why were these tasks removed? If I'm not wrong, we need them to generate the `dist` we publish to **npm**.

Reply:
`prepublishOnly` is referenced on `package.json`

Reply:
I think I missed the reference to `dist`.  If at https://github.com/underscopeio/sinon-mongoose/blob/master/package.json#L14 we change "main": "dist/index.js" to "main": "lib/index.js" I think it should work wihtout having to generate a `dist` and still publish to npm.  Right?

Reply:
I see, babel is doing nothing but adding some spaces.